ui/tests/acceptance/token-test.js

```diff
@@ -1,10 +1,8 @@
+import $ from 'jquery';
```

Comment:
Is this kinda thing on purpose? Are they the exact same?

Reply:
Are what the exact same?

Reply:
`$` from `jquery` and `Ember.$`. If you look at the rest of the diff it swaps them.

Reply:
Ah, yeah, they're the same. Ember tacked $ until the Ember global so there was only one global to deal with. Also to open the possibility of `Ember.$` being swapped out for Zepto, or some other jquery-like interface. 

Now that everything is modules, it makes sense to import it directly from `jquery` rather than `@ember/jquery`.

Still kind of a gotcha, since components have a `$` property, which is a scoped jquery selector. Could get confusing if someone did something like `import $ from zepto` in their app.